prime-router/metadata/hl7_mapping/ORU_R01/base/observation-request.yml

```diff
@@ -145,17 +145,17 @@ elements:
   - name: result-interpreter
     resource: '%resource.resultsInterpreter.resolve()'
     constants:
-      cnnFieldPath: '%{fieldPath}-32-1'
-    schema: ../../common/datatype/cnn-composite-number-name
+      hl7CNNField: '%{fieldPath}-32-1'
+    schema: ../../Resources/Practitioner/CNN
 
   - name: technician
     resource: '%resource.performer.resolve().where(extension("http://hl7.org/fhir/StructureDefinition/event-performerFunction").value.coding.code = "SPRF").practitioner.resolve()'
     constants:
-      cnnFieldPath: '%{fieldPath}-34-1'
-    schema: ../../common/datatype/cnn-composite-number-name
+      hl7CNNField: '%{fieldPath}-34-1'
+    schema: ../../Resources/Practitioner/CNN
```

Comment:
This is the specific temporary mapping that is likely causing the HCI integration test failure. This should be corrected when the final NDL mapping is complete.